attribution/index.html

```diff
@@ -40,11 +40,11 @@
                     <img class="hero-img" src="../assets/img/heros/hero-attribution.jpg" alt="Black Hole">
                 </div>
                 <div class="hero-content">
-                    <span class="about-hero-text">
+                    <div class="about-hero-text">
```

Comment:
We aren't using this class, should we just get rid of it if that's the case?

Reply:
this looked like it was changed in the accessibility branch

Reply:
This branch was created this morning after pulling from main, so this class was still in there. It might not be used (which I just found out as I was fixing an error). I recommend merging this one then creating a new branch focused just on removing this class from all the pages.

Reply:
Got it. Sounds good

Reply:
I take that back...the branch was created yesterday with a pull first. But, I still recommend the same procedure because the branch I created this morning for accessibility still shows this class.